metar/Metar.py

```diff
@@ -412,17 +427,25 @@ def __init__(self, metarcode, month=None, year=None, utcdelta=None,
           if pattern == REMARK_RE or self.press:
               while code:
                   for pattern, handler in self.remark_handlers:
-                      if debug: print(handler.__name__,":",code)
+                      if debug:
+                          _logger.debug("%s: %s", handler.__name__, code)
                       m = pattern.match(code)
                       if m:
-                          if debug: _report_match(handler,m.group())
+                          if debug:
+                              _report_match(handler, m.group())
                           handler(self,m.groupdict())
                           code = pattern.sub("",code,1)
                           break
 
       except Exception as err:
-          raise ParserError(handler.__name__+" failed while processing '"+
-                            code+"'\n"+" ".join(err.args))
+          message = (
```

Comment:
sorry to go back and forth here, but can we please back this `strict` change out as well and isolate to another commit.

Reply:
done, but then i accidentally left in my editor's trailing whitespace deletion 😢

Reply:
Thanks for your patience, up to you on this one, if fixing is hard, no worries.

Reply:
let's leave them then. they're harmless and it will mean all the fewer changes for the big black out